osgi-bundles/bundles/jruby/pom.xml

```diff
@@ -52,11 +52,13 @@
         <dependency>
             <groupId>org.kill-bill.billing</groupId>
             <artifactId>killbill-api</artifactId>
+            <version>0.15-SNAPSHOT</version>
```

Comment:
Tip: when developing against multiple projects locally, bump the SNAPSHOT versions directly in `killbill-oss-parent`. That way, you only need to bump one version, i.e. the one of `killbill-oss-parent` in the main pom.xml of each project.

(these changes will need to be reverted before merging)